packages/design-picker/src/utils/designs.ts

```diff
@@ -0,0 +1,28 @@
+import { addQueryArgs } from '@wordpress/url';
+import type { Design, DesignPreviewOptions } from '../types';
+
+export const getDesignPreviewUrl = (
+	design: Design,
+	options: DesignPreviewOptions = {}
+): string => {
+	const { recipe } = design;
+
+	let url = addQueryArgs( 'https://public-api.wordpress.com/wpcom/v2/block-previews/site', {
+		theme: recipe?.theme,
+		pattern_ids: recipe?.patternIds?.join( ',' ),
+		language: options.language,
+		viewport_height: 700,
+	} );
+
+	const siteTitle = options.siteTitle || design.title;
+	if ( siteTitle ) {
```

Comment:
No we can't; that's exactly the problem with `addQueryArgs()`, if we pass the title to it, it will encode the already encoded `(` (into `%28`) and then we will get the wrong result.

Reply:
Ooops you're right. I forgot it will encode `%` to `%25`